physical_property/logD/calculate_logD/README.md

```diff
@@ -0,0 +1,6 @@
+## What's here
+
+- [`calc_logD.nb`](calc_logD.nb) - Wolfram Mathematica notebook used to calculate distribution coefficients (log *D*<sub>7.4</sub>). The notebook takes the partition coefficients (log *P*) and acidity constants from the SAMPL7 log *P* and pK<sub>a</sub> submission collection files and calculates the distribution coefficients according to the model described in https://doi.org/10.1007/s10822-016-9954-8. Because the charge state information for the protonated and deprotonated species isn't available for the individual models, eq. 4 for acids was used for all compounds. This choice was made because of the broad consensus of different pK<sub>a</sub> models that had also submitted macrostate pK<sub>a</sub> values including the charge state information. The distribution coefficients are only calculated for combinations of ranked models submitted by participants from the same participant organization. Notebook created by Nicolas Tielker.
```

Comment:
You should make clear whether this is the notebook used to generate the analysis/plots provided here, or a notebook which Tielker generated which was then used as the basis for our anaysis, or what.

Reply:
Updated👍